docs/pages/setup/security/hardened-container-images.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Hardened Container Images
+description: How to run Teleport in hardened container images
+---
+
+Customers running in heavily regulated or compliance environments may be
+interested in running Teleport within a minimal or hardened container image.
+This may be customers running in FedRAMP environments or customers that have a
+requirement to meet CIS Hardened Image guidelines.
+
+We have plans to publish harded container images in the second half of 2022.
+
+In the meantime, customers can construct a hardened image that meets those
+requirement by running Teleport within `distroless` container image using the
+example `Dockerfile` below.
+
+```code
+# Use multistage builds to extract libgcc_s.so and Teleport binaries.
+FROM quay.io/gravitational/teleport:(=teleport.version=) AS teleport-base
+FROM gcr.io/distroless/cc-debian11 AS cc
+
+# Run Teleport within a base-debian11 distroless container image.
+FROM gcr.io/distroless/base-debian11
```

Comment:
Why debian? its one of the distros which typically contains more vulnerabilities. Even the distroless version at the time of writing contains 13 (scanned with trivy).

If the goal is to provide an image more suitable for federal / secure environments, Amazon linux, Alpine, or perhaps even RHEL would be better choices.

Amazon linux and RHEL are also more suited for FIPS.

Could we get install instructions for different distros?